integration/rocket-chat/main.js

```diff
@@ -537,17 +537,17 @@ function SmartiWidget(element,_options) {
 
                     //map to dbsearch results TODO should be configurable
                     var docs = $.map(data.response.docs, function(doc) {
-                        return {
+                        /*return {
```

Comment:
ah, sorry, this should not go in the request. With the new query providers this part will be changed. For the moment i will revert this change. THX